packages/neon-api/package-lock.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
this seems like a temporary file based on your setup. I don't think this should be added